app/templates/components/settings/contact-info-section.hbs

```diff
@@ -0,0 +1,11 @@
+<form class="ui form" {{action 'submit' on = 'submit'}}>
+  <div class="field">
+    <label>Email</label>
+    <input type="text" name="email" placeholder="Email">
```

Comment:
Use the ember `input` helper so that the value is binded to the object.
`{{input type='text' name='email' placeholder='john.doe@example.com' value=email}}`